src/_variables.scss

```diff
@@ -183,7 +183,7 @@ $layout-drawer-border-color: unquote("rgb(#{$palette-grey-300})") !default;
 $layout-text-color: unquote("rgb(#{$palette-grey-800})") !default;
 $layout-drawer-navigation-color: #757575 !default;
 $layout-drawer-navigation-link-active-background: unquote("rgb(#{$color-light-contrast})") !default;
-$layout-drawer-navigation-link-active-color: unquote("rgb(#{$color-primary})") !default;
+$layout-drawer-navigation-link-active-color: unquote("rgb(#{$palette-grey-300})") !default;
```

Comment:
I don't know tbh. We have tons of theme colors and the spec isn't exactly clear on which color to use. Since this is the *drawer* it should work regardless of theme color unless you do a dark drawer color (like in the dashboard template.) I simply pulled the menu item's hover color since it seemed like the best fit for what we have compared to the spec pages.

Ideally, this is at least a good baseline for the single-color drawer we provide until the spec is updated with a more direct solution.